bgpd/bgp_packet.c

```diff
@@ -1385,6 +1407,26 @@ static int bgp_keepalive_receive(struct peer *peer, bgp_size_t size)
 	return Receive_KEEPALIVE_message;
 }
 
+static int bgp_refresh_stalepath_timer_expire(struct thread *thread)
+{
+	struct peer *peer;
+	afi_t afi;
+	safi_t safi;
+
```

Comment:
how do you know which afi and safi is linked to that event.
my understanding of the capability is that it is a per afi/Safi capability
as such, route refresh can happen based on afi/Safi, not all.

Reply:
thanks.